StarWars.Core/ViewModels/MainViewModel.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using MvvmCross.Commands;
+using MvvmCross.Navigation;
+using MvvmCross.ViewModels;
+using StarWars.Core.Models;
+using StarWars.Core.Services.Abstract;
+
+namespace StarWars.Core.ViewModels;
+
+public sealed class MainViewModel : MvxViewModel
+{
+    private readonly IMvxNavigationService _navigationService;
+    private readonly ISwapiService _swapiService;
+    public IMvxCommand<SwapiPersonModel> SelectPersonCommand { get; }
+
+    public MainViewModel(IMvxNavigationService navigationService, ISwapiService swapiService)
+    {
+        _navigationService = navigationService;
+        _swapiService = swapiService;
+        
+        SelectPersonCommand = new MvxCommand<SwapiPersonModel>(async person =>
```

Comment:
You can use here IMvxAsyncCommand
SelectPersonCommand = new MvxAsyncCommand<SwapiPersonModel>(OpenDetailsAsync);